wd_core.au3

```diff
@@ -166,6 +174,14 @@ Global $_WD_DefaultTimeout = 10000 ; 10 seconds
 Global $_WD_WINHTTP_TIMEOUTS = True
 Global $_WD_HTTPTimeOuts[4] = [0, 60000, 30000, 30000]
 Global $_WD_HTTPContentType = "Content-Type: application/json"
+
+Global $_WD_SupportedBrowsers[][$_WD_BROWSER__COUNTER] = _

```

Comment:
Please add 
```autoit
Global Enum $_WD_BROWSERID_Chrome, $_WD_BROWSERID_FireFox, $_WD_BROWSERID_Edge, $_WD_BROWSERID_Opera
```
```
It will be used for example in wd_capabilities.au3

Reply:
@mlipok Please elaborate with an example of how this would look in wd_capabilities.au3

Reply:
aftere rethinking it will not be needed